pkg/kubelet/rkt/rkt.go

```diff
@@ -830,6 +830,13 @@ func (r *Runtime) generateRunCommand(pod *api.Pod, uuid string) (string, error)
 
 	var hostname string
 	var err error
+
+	// Overlay fs is not supported for SELinux yet.
```

Comment:
Interesting, I didn't realize anyone had put the effort in to make overlayfs work with SELinux.  TIL!